app/frontend/components/pages/delegations/shelleyBalances.tsx

```diff
@@ -24,121 +24,128 @@ const ShelleyBalances = ({
     },
   } = useActiveAccount()
   return (
-    <div className="rewards card">
```

Comment:
changes looks serious, but here is just wrapped whole content inside container.

Similarly in `sendAdaPage.tsx`,